### PR TITLE
updated priya info

### PR DESCRIPTION
**Pull request by priyasilverstein**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -38,11 +38,11 @@ people:
   position: TEST
```
Comment (Remi-Gau) [inline _config.yml]: @priyasilverstein Do you want to add that you are doing a post-doc at the moment in front of the position section?

**Review by Remi-Gau — CHANGES_REQUESTED** [review]
One additional thing you might want to add. :-p

**Comment by priyasilverstein** [comment]
Oooh good idea hehe sorry, will do when I’m avec laptop

**Review by Remi-Gau — COMMENTED** [review]
LGTM: Looks good to me

**Review by Remi-Gau — APPROVED** [review]

**Pull request merged**